scripts/setup-gateway.sh

```
@@ -0,0 +1,587 @@
+    log_debug "  Using local manifest: $maas_networking_dir"
+    kustomize build "$maas_networking_dir" | \
+      CLUSTER_DOMAIN="$CLUSTER_DOMAIN" CERT_NAME="$CERT_NAME" envsubst '$CLUSTER_DOMAIN $CERT_NAME' | \
+      kubectl apply --server-side=true -f -
+  elif [[ "$DISCONNECTED" == "true" ]]; then
+    log_error "DISCONNECTED=true but local manifest not found: $maas_networking_dir"
+    exit 1
+  else
+    # Fallback: fetch from GitHub
+    log_debug "  Local manifest not found, fetching from GitHub..."
+    kustomize build "https://github.com/opendatahub-io/models-as-a-service.git/deployment/base/networking/maas?ref=main" | \
+      CLUSTER_DOMAIN="$CLUSTER_DOMAIN" CERT_NAME="$CERT_NAME" envsubst '$CLUSTER_DOMAIN $CERT_NAME' | \
```

Shellcheck (0.11.0)

[info] Expressions don't expand in single quotes, use double quotes for that.

(SC2016)